### PR TITLE
Add collection view for menu view

### LINKED ISSUE #8: Create Menu view using IGListKit
**Describe the solution you'd like**
A menu display view controller. 

**Designs for said solution**
Following design from Zeplin (when available)

### PR DESCRIPTION
**Pull request by jaewonsim**
Resolves #8

- Add IGListKit-based menu view
- Navigation controller to be implemented

### CODE REVIEW

File: acai-ios/CollectionViewCells/MenuCollectionViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        setUpConstraints()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Constraint setup
+    private func setUpConstraints() {
+        let leadingOffset: CGFloat = 24
```
Comment (YKo20010) [inline acai-ios/CollectionViewCells/MenuCollectionViewCell.swift:59]: might want to put constants at the top along with view vars

**Comment by doctor3w** [comment]
Merged with master and updated design

File: acai-ios/MenuViewController.swift [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        loadMenu()
+    }
+
+    private func loadMenu() {
+        // TODO: get menu from endpoint
```
Comment (cornell-appdev) [inline acai-ios/MenuViewController.swift:63]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/cuappdev/acai-ios/blob/b53abc5adb28ebe817eeead00342a502917dda75/acai-ios/MenuViewController.swift#L63"></span>TODOs should be resolved (get menu from endpoint).<br />
<code>todo</code> <code>MenuViewController.swift:63</code></td>
    </tr>
  </tbody>
</table>

**Comment by cornell-appdev** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">SwiftLint not installed, download from https://github.com/realm/SwiftLint</td>
    </tr>
  </tbody>
</table>

### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
MenuViewController.swift | 63 | TODOs should be resolved (get menu from endpoint). (todo)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: acai-ios/CollectionViewCells/MenuCollectionViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    
+    // MARK: View vars
+    var imageView: UIImageView!
+    var titleLabel: UILabel!
+    var ingredientsLabel: UILabel!
+    var detailStackView: UIStackView! // include title and ingredients
+    var bottomSeparator: UIView!
+
+//    var additionalPricingLabel: UILabel!
+//    var selectButton: UIButton!
```
Comment (kevinchan159) [inline acai-ios/CollectionViewCells/MenuCollectionViewCell.swift]: remove if unnecessary

File: acai-ios/CollectionViewCells/MenuCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+//        additionalPricingLabel.font = UIFont.systemFont(ofSize: 12, weight: .medium)
+//        additionalPricingLabel.textColor = .black
+//        contentView.addSubview(additionalPricingLabel)
+//
+//        selectButton = UIButton()
+//        selectButton.layer.cornerRadius = 8
+//        selectButton.setTitle("Select", for: .normal)
+//        selectButton.backgroundColor = UIColor(red: 1.0, green: 156.0 / 255.0, blue: 29.0 / 255.0, alpha: 1.0)
+//        selectButton.tintColor = .white
+//        contentView.addSubview(selectButton)
```
Comment (kevinchan159) [inline acai-ios/CollectionViewCells/MenuCollectionViewCell.swift]: remove if unnecessary

File: acai-ios/CollectionViewCells/MenuCollectionViewCell.swift [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    
+    // MARK: Constraint setup
+    private func setUpConstraints() {
+        let leadingOffset: CGFloat = 24
+        let trailingInset: CGFloat = 72
+//        let selectButtonWidth: CGFloat = 84
+        let imageViewWidthHeight: CGFloat = 64
+        let imageViewStackViewHorizontalSpacing: CGFloat = 24
+        let bottomSeparatorHeight: CGFloat = 1.0
+//        let stackViewSelectButtonHorizontalSpacing: CGFloat = 17
```
Comment (kevinchan159) [inline acai-ios/CollectionViewCells/MenuCollectionViewCell.swift]: remove if unnecessary

File: acai-ios/CollectionViewCells/MenuCollectionViewCell.swift [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+            make.centerY.equalToSuperview()
+            make.leading.equalToSuperview().inset(leadingOffset)
+            make.width.height.equalTo(imageViewWidthHeight)
+        }
+        
+        detailStackView.snp.makeConstraints { make in
+            make.leading.equalTo(imageView.snp.trailing).offset(imageViewStackViewHorizontalSpacing)
+            make.centerY.equalToSuperview()
+            make.trailing.equalToSuperview().inset(trailingInset)
+//            make.trailing.equalTo(selectButton.snp.leading).offset(-stackViewSelectButtonHorizontalSpacing)
```
Comment (kevinchan159) [inline acai-ios/CollectionViewCells/MenuCollectionViewCell.swift]: same

File: acai-ios/CollectionViewCells/MenuCollectionViewCell.swift [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+        bottomSeparator.snp.makeConstraints { make in
+            make.leading.trailing.bottom.equalToSuperview()
+            make.height.equalTo(bottomSeparatorHeight)
+        }
+        
+//        selectButton.snp.makeConstraints { make in
+//            make.trailing.equalToSuperview().inset(trailingInset)
+//            make.width.equalTo(selectButtonWidth)
+//            make.centerY.equalToSuperview()
```
Comment (kevinchan159) [inline acai-ios/CollectionViewCells/MenuCollectionViewCell.swift]: same

**Review by kevinchan159 — DISMISSED** [review]
lgtm

**Pull request merged**